src/main/java/rwg/config/ConfigRWG.java

```diff
@@ -16,6 +16,7 @@ public class ConfigRWG {
     public static boolean generateVillages = true;
     public static boolean generateUndergroundLakes = true;
     public static boolean generateUndergroundLavaLakes = true;
+    public static boolean suptcbig = true;
```

Comment:
Please use readable names for fields. No one wants to decipher what `suptcbig` means.
```suggestion
    public static boolean generateLargeThaumcraftBiomes = true;
```